doc/user-docs/index.md

```diff
@@ -67,16 +67,15 @@ Not currently supported "out of the box". See issue [#1060](https://github.com/m
 
 ## Configuring Windows Terminal
 
-At the time of writing all Windows Terminal settings are managed via a json file.
+All Windows Terminal settings are currently managed using the `profiles.json` file, located within `$env:LocalAppData\Packages\Microsoft.WindowsTerminal_<randomString>/RoamingState`.
```

Comment:
The string isn't random... 8wekyb3d8bbwe is the encoded version of Microsoft Corporation for store signing purposes.

You can put the actual string there and it should be stable indefinitely.

Reply:
Oh, I see. You just relocated this text. Well if you could fix that while you're here, it might be nice.

Reply:
Ah, that does make things easier to document. Added commit for `8wekyb3d8bbwe`